src/main/java/io/SchoolManagementSystem/Attendance.java

```diff
@@ -0,0 +1,9 @@
+package io.SchoolManagementSystem;
+
+public class Attendance {
```

Comment:
Attendance is not pointed by any other class, which gives the impression that It can be instantiated anywhere.